QuizAppNew/View/HomeView.swift

```diff
@@ -68,4 +69,24 @@ struct HomeView: View {
             })
         }
     }
+    
+    func nazmul() {
+    
+        if let path = Bundle.main.path(forResource: "data", ofType: "json") {
+            do {
+                  let data = try Data(contentsOf: URL(fileURLWithPath: path), options: .mappedIfSafe)
+                let _posts = try? JSONDecoder().decode([QuestionModel].self, from: data)
+    
+                for item in _posts! {
+                    print(item.qustion)
+                }
+    
+            } catch {
+                   // handle error
+                  //      print("error")
```

Comment:
what if you getting the error?  how do you know?

Reply:
error handler.  if json file was not load then error sms will print ..

Reply:
I don't see any print, how you understand  just print "error" message.  what is the actual issue without  dealing with catch block.

Reply:
I don't see any print, btw how do you understate just  **print("error")**?  what is the actual cause for the error occurring? how do you deal with that after getting an error?

Reply:
print("error") it was wrong..

Reply:
aita dia tumi ki vujba. ki error khaiche keno catch arche. ki jamela hochey code er bitor.

Reply:
catch kokhon ashe catch ki vabe error ditect kore